presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorAccessControl.java

```diff
@@ -287,4 +293,14 @@ default void checkCanRevokeTablePrivilege(ConnectorTransactionHandle transaction
     {
         denyRevokeTablePrivilege(privilege.toString(), tableName.toString());
     }
+
+    /**
+     * Check if identity is allowed to select from the specified columns in a table.
+     *
+     * @throws com.facebook.presto.spi.security.AccessDeniedException if not allowed
+     */
+    default void checkCanSelectFromColumns(ConnectorTransactionHandle transactionHandle, Identity identity, SchemaTableName tableName, Set<String> columnNames)
```

Comment:
Should we put this next to the table permissions, rather than adding it at the end? Keeping similar methods next to each other is helpful.

Same question for other interfaces and implementations.